staff.html

```diff
@@ -0,0 +1,119 @@
+                <h3 class="text-lg">Sir monkey the 3rd#1507</h3>
+                <p>Owner</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+                <h3 class="text-lg">Jaronline#1503</h3>
+                <p>Owner <br> Head developer</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+                <h3 class="text-lg">SPITFIRE#4794</h3>
+                <p>Board of Director <br> Website developer</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+                <h2 class="text-lg">Luke M#6078</h2>
+                <p>Board of Director</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+              <h2 class="text-lg">corn pops#0949</h2>
+              <p>Board of Director</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+              <h2 class="text-lg">Luke M#6078</h2>
+              <p>Board of Director</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+              <h2 class="text-lg">Dumbo#8163</h2>
+              <p>Board of Director</p>
+            </div>
+
+        </div>
+    </main>
+    <footer class="px-30 pb-1 pt-3 bg-primary space-y-4">
+        <div class="flex justify-between">
+            <ul class="space-y-1">
+                <!-- <li><h4>Information</h4></li> -->
+                
+                <li>
+                    <a href="privacy">Privacy Policy</a>
+                </li>
+                <li>
+                    <a href="terms">Terms of Service</a>
+                </li>
+            </ul>
+            <ul class="space-y-1">
+                <!-- <li><h4>Information</h4></li> -->
+                <!-- <li>
+                    <a href="rules">Rules</a>
+                </li>
+                <li>
+                    <a href="blog">Announcements</a>
+                </li>
+                <li>
+                    <a href="tutorial">Tutorials</a>
+                </li> -->
+            </ul>
+            <ul class="space-y-1">
+                <!-- <li><h4>Information</h4></li> -->
+                <!-- <li>
+                    <a href="apply">Apply</a>
+                </li>
+                <li>
+                    <a href="appeal">Ban Appeal</a>
+                </li>
+                <li>
+                    <a href="suggest">Suggestions</a>
+                </li> -->
+            </ul>
+            <ul class="space-y-1">
+                <!-- <li><h4>Information</h4></li> -->
+                <li>
+                    <a href="commands">Commands</a>
+                </li>
+                <li>
+                    <a href="https://beta.statcord.com/bot/709705136259334296" target="_blank" rel="noopener noreferrer">Statistics</a>
+                </li>
+            </ul>
+        </div>
+        <div class="flex justify-center gap-1">
+            <img src="https://github.com/PixelPizza/Images/blob/main/b3ae66a1d6f601eb55114fbd172ad973.png?raw=true">
```

Comment:
Could you make this into a local image? so it doesn't have to load it from github

Reply:
Nvm, I'll do it myself for all of them

Reply:
Ok you can do it if you want